rbtree.c

```diff
@@ -4,6 +4,11 @@
  */
 #include <ruby/ruby.h>
 #include <ruby/version.h>
+
+#ifndef RUBY_API_VERSION_CODE
+#define RUBY_API_VERSION_CODE (RUBY_API_VERSION_MAJOR*10000+RUBY_API_VERSION_MINOR*100+RUBY_API_VERSION_TEENY)
+#endif
```

Comment:
This part is because `RUBY_API_VERSION_CODE` wasn't added until Ruby 1.9, and the test suite indicates that 1.8 compatibility is still considered in this gem.

Reply:
Did this by adjusting the if guard instead.

Reply:
Did this by adjusting the if guard instead.